File: output/llama_4_maverick/avatar/algo_based_translation/java/python/successful_translation/codeforces_621_A.py

n = int(input())
numbers = list(map(int, input().split()))
odd_numbers = []
total_sum = 0
for num in numbers:
    if num % 2 == 0:
        total_sum += num
    else:
        odd_numbers.append(num)
odd_numbers.sort()
total_sum += sum(odd_numbers)
if len(odd_numbers) % 2 != 0:
    total_sum -= odd_numbers[0]
print(total_sum)

File: output/deepseek_r1/avatar/algo_based_translation/python/java/codeforces_621_A.java

import java.util.Scanner;
import java.util.Arrays;

public class codeforces_621_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        scanner.nextLine();
        int[] lst = new int[n];
        String[] inputs = scanner.nextLine().split(" ");
        for (int i = 0; i < n; i++) {
            lst[i] = Integer.parseInt(inputs[i]);
        }
        Arrays.sort(lst);
        for (int i = 0; i < n / 2; i++) {
            int temp = lst[i];
            lst[i] = lst[n - 1 - i];
            lst[n - 1 - i] = temp;
        }
        int od = 0;
        for (int num : lst) {
            if (num % 2 != 0) {
                od++;
            }
        }
        int sum = 0;
        boolean ok = false;
        for (int current : lst) {
            if (current % 2 == 0) {
                sum += current;
            } else {
                if (ok || od > 1) {
                    sum += current;
                    od--;
                    ok = !ok;
                }
            }
        }
        System.out.println(sum);
    }
}

